about.html

```diff
@@ -0,0 +1,281 @@
+            <article class="history-bottom">
+                <div class="slide-content">
+                   <div class="card-wrapper">
+                        <div class="cardd">
+                            <div class="image-content">
+                                <span class="overlay"></span>
+                                <div class="card-image">
+                                    <img src="./assets/images/cvimage2.JPG" alt="" class="card-img">
+                                </div>
+                            </div>
+                            <div class="card-content">
+                                <h3 class="name">Mike Tyson</h3>
+                                <p class="description">An owesome content creator, web developer and alumni at BKE COURSE</p>
+                                <button class="bdn">view more</button>
+                            </div>
+                        </div>
+                   </div> 
+                </div>
+
+                <div class="slide-content">
+                    <div class="card-wrapper">
+                         <div class="cardd">
+                             <div class="image-content">
+                                 <span class="overlay"></span>
+                                 <div class="card-image">
+                                     <img src="./assets/images/cvimage2.JPG" alt="" class="card-img">
+                                 </div>
+                             </div>
+                             <div class="card-content">
+                                 <h3 class="name">Mike Tyson</h3>
+                                 <p class="description">An owesome content creator, web developer and alumni at BKE COURSE</p>
+                                 <button class="bdn">view more</button>
+                             </div>
+                         </div>
+                    </div> 
+                 </div>
+
+                 <div class="slide-content">
+                    <div class="card-wrapper">
+                         <div class="cardd">
+                             <div class="image-content">
+                                 <span class="overlay"></span>
+                                 
+                                 <div class="card-image">
+                                     <img src="./assets/images/cvimage2.JPG" alt="" class="card-img">
+                                 </div>
+                             </div>
+                             <div class="card-content">
+                                 <h3 class="name">Mike Tyson</h3>
+                                 <p class="description">An owesome content creator, web developer and alumni at BKE COURSE</p>
+                                 <button class="bdn">view more</button>
+                             </div>
+                         </div>
+                    </div> 
+                 </div>
+            </article>
+            
+        </article>
+    </section>
```

Comment:
- Good job done so far, but please note that as much as you are required to personalize the content of your project, it is still important to stick to the design layout, spacing, colors etc. There are two missing sections in your project, the Logo section and the `previous summit section`. Please, follow strictly the layout of the design template to give confidence to recruiters in your ability to replicate exact business requirements. Please fix this 🙏 
Your project:
-Please design for both desktop and mobile versions accordingly 👍 👍 
<img width="629" alt="erergergke" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/03a1449c-c2e3-4d4a-9c50-939a702df0ac">

Design template:
<img width="557" alt="wegqeg" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/e9a4dfc6-1173-435b-bda3-fa53d607d228">

- Also, on the mobile version of the About page, the `partner` section is missing, please include this section in your design 🙏 
Design template:
<img width="318" alt="rgqwg3" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/669e834c-37e9-4b27-9a26-eae9234dc9a1">